### ISSUE TITLE
output parameter doesn't seem to work for mvt-android check-backup 

### ISSUE THREAD

**Issue by bkgrrl00** [mvt-project/mvt#366]
I was able to extract SMS and MMS messages from my Android device and check them with the `mvt-android check-backup` command. However, the `--output` parameter did not result in an output file. 

This is what I used (based on the MVT documentation):

`mvt-android check-backup --output /path/to/results/output.txt /path/to/results/SMSbackup.ab`

**Comment by DonnchaC** [comment]
The output path should be a directory where MVT can store JSON files containing the module results.

Can you check again with a directory path on the latest version of MVT and let us know if the problem still exists?

**Comment by bkgrrl00** [comment]
I want to retest this, but I don't currently have MVT installed, and I'm having problems re-installing it. I'll retest once I am able to re-install.

(I've posted a separate issue [here](https://github.com/mvt-project/mvt/issues/374) re the re-installation problem.)

**Comment by Forchapeatl** [comment]
Hello @bkgrrl00 , indeed the output path should be a directory and not a file. Please try this command instead 
`mvt-android check-backup --outp output backup.ab` 
If an output directory does not exsit , MVT will create one for you
![image](https://github.com/mvt-project/mvt/assets/24577149/b2a80ad6-ec7c-452f-9015-260126777ef2)

The lo file will appear in the output directory once the backup is done.
![image](https://github.com/mvt-project/mvt/assets/24577149/fbf68b4d-f951-4d04-90c4-7ae506505971)

**Comment by Te-k** [comment]
Thanks @Forchapeatl , it sounds solved to me, I am going to close that issue but feel free to reopen it @bkgrrl00 if you encounter any new issue with that

**Comment by Forchapeatl** [comment]
My pleasure
